app/views/provider/admin/api_docs/show.html.erb

```diff
@@ -12,11 +13,31 @@
   <p class="InfoBox InfoBox--notice"><%= link_to 'Create an access token', provider_admin_user_access_tokens_path %> to authenticate against the Account Management API, the Analytics API and the Billing API.</p>
 <% end %>
 
-<div class='api-docs-wrap'></div>
-<script>
-$(function(){
-   ThreeScale.APIDocs.account_type = 'provider';
-   ThreeScale.APIDocs.host = "<%=j base_url.gsub(%r{/$}, '') %>";
-   ThreeScale.APIDocs.init([]);
-});
-</script>
+<div class='api-docs-wrap' id = "analytics-api-docs"></div>
+<div class='api-docs-wrap' id = "billing-api-docs"></div>
+<div class='api-docs-wrap' id = "service-management-api-docs"></div>
+<div class='api-docs-wrap' id = "account-management-api-docs"></div>
+<div class='api-docs-wrap' id = "policy-registry-api-docs"></div>
+
+<script type="text/javascript">
+  $(function () {
+    let path = []
+    var response = '';
+    $.ajax({ type: "GET",
+             url: "<%= j api_docs_services_path %>",
+             async: false,
+             success : function(text)
+             {
+                 response = text.apis;
+                 var len = response.length
+                 for (var i = 0; i < len; i++)
+                  path.push(response[i].path)
+             }
+    });
+    path.forEach(function (url) {
```

Comment:
`path` will be empty at this point, b/c `path` is filled by the `success` callback when the ajax call gets a response. This should be moved inside the `success` callback.

Reply:
I've seen that the `async` option in the ajax call is set to `false`, this makes it all synchronous, so what I said above is wrong. Anyway, not sure it's the best idea. Opinions?